iceberg/iceberg-catalog/src/main/java/org/apache/iceberg/hive/HiveTableOperations.java

```diff
@@ -270,7 +275,9 @@ protected void doCommit(TableMetadata base, TableMetadata metadata) {
 
         LOG.error("Cannot tell if commit to {}.{} succeeded, attempting to reconnect and check.",
             database, tableName, e);
-        commitStatus = checkCommitStatus(newMetadataLocation, metadata);
+        commitStatus =
+                BaseMetastoreOperations.CommitStatus.valueOf(
```

Comment:
is it possible to import BaseMetastoreOperations.CommitStatus and avoid this long naming

Reply:
No.  BaseMetastoreTableOperations also has the enum `CommitStatus ` which has been deprecated, we need to specify the full prefix to use `BaseMetastoreOperations.CommitStatus `
https://github.com/apache/iceberg/blob/8e9d59d299be42b0bca9461457cd1e95dbaad086/core/src/main/java/org/apache/iceberg/BaseMetastoreTableOperations.java#L291-#L293

Reply:
are we using both enums at the same time? since it was deprecated why not drop it?

Reply:
👍